scripts/common.bash

```diff
@@ -13,15 +13,21 @@ function for_each_unittest()
 	local check
 	local accel
 	local timeout
+	local run_under_efi
```

Comment:
We have an `EFI_RUN` environment variable to `x86/run` and `x86/efi/run`:

https://github.com/marc-orr/KVM-Unit-Tests-dev-fork/blob/c2fcb520ad5fd6df1861d47296722f179d41d579/x86/run#L47-L56

https://github.com/marc-orr/KVM-Unit-Tests-dev-fork/blob/c2fcb520ad5fd6df1861d47296722f179d41d579/x86/efi/run#L55-L63

Maybe we can unify the usage of `run_under_efi` and `EFI_RUN`?

I guess the easy step is to rename the `EFI_RUN` to `run_under_efi` because `EFI_RUN` only occurs twice.

Then pass the `run_under_efi` as an environment or export it, so the `x86/run` and `x86/efi/run` can directly use it?